jekyll/_docs/enterprise/gpu-configuration.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: enterprise
+section: enterprise
+title: "GPU Builder Installation"
+category: [advanced-config]
+order: 6
+description: "How to install CircleCI Enterprise LXC on Ubuntu 14.04"
+---
+
+#GPU Install Instructions 
+
+The following steps are the same whether you have an LXC based builder or a Docker based builder. They diverge later on.
+
+1.)`wget https://developer.nvidia.com/compute/cuda/8.0/prod/local_installers/cuda-repo-ubuntu1404-8-0-local_8.0.44-1_amd64-deb`
```

Comment:
This numbered list, and the backticks are broken. Page isn't rendering right.
![broken-page](https://cloud.githubusercontent.com/assets/6017470/22810179/3a33b410-ef05-11e6-9f7b-689e71ea6947.jpg)